### DISCUSSION TITLE
"1 issue needs help" stays even after removing the label

### DISCUSSION THREAD

**Discussion by ussserrr** [community/community#22715] [New to GitHub]
<p>I’ve accidentally added the “help wanted” label to the issue in my repository and soon removed it. Though, the corresponding mark is still there even if I don’t have any appropriate issues anymore (about 3 days have passed already). What can I do?</p>
<p><img src="https://user-images.githubusercontent.com/20915411/181072689-1ebeec4f-bd4e-48a0-bdff-45d96094abf6.png" alt="1.png" data-base62-sha1="4b45ybKhvaCIK8cz7PBgoTZhsi1" width="430" height="119"> <img src="https://user-images.githubusercontent.com/20915411/181072690-e0a77db3-18a2-4a56-936b-7ad3f35c7c2f.png" alt="2.png" data-base62-sha1="bz9yH52kIOurcrmkBLx9wdnsJa6" width="496" height="456"></p>

**Comment by ghost** [comment]
<p>Hi <a class="mention" href="/ussserrr">@ussserrr</a> - thanks for reaching out in our community forums. </p>
<p>The status of issues can get stuck in the issue index sometimes. Usually this resolves itself, but we can also manually re-index the repo index for you too. </p>
<p>I have gone ahead and done this for you. If this doesn’t resolve the problem let me know! </p>

**Comment by ussserrr** [comment]
<p>Thanks! Do you see a label now (<a href="https://github.com/ussserrr?tab=repositories" rel="nofollow noopener">here</a> for example)? Cause I am for sure. Probably, something other should be done…</p>

**Comment by ussserrr** [comment]
<p>OK, so now I close this issue completely and that label is still present.</p>

**Comment by ussserrr** [comment, marked as answer]
<p>The label is gone now, finally. After almost 4 months :upside_down_face:. Not sure, what happened though. Maybe folks from the GitHub refresh their DBs recently.</p>
